quodlibet/commands.py

```diff
@@ -323,7 +323,7 @@ def _rating(app, value):
             change = (1 / RATINGS.number)
         if value[0] == '-':
             change = -change
-        rating = song["~#rating"] + change
+        rating = song.get("~#rating", RATINGS.default) + change
```

Comment:
I think `song("~#rating")` should work here and not duplicate that logic

Reply:
Thanks.  I force pushed a new version